inc/c_util/tarray_ll.h

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef TARRAY_LL_H
+#define TARRAY_LL_H
+
+#ifdef __cplusplus
+#include <cinttypes>
+#else
+#include <inttypes.h>
+#endif
+
+#include "c_pal/interlocked.h"
+
+#include "c_util/thandle_ll.h"
+
+#include "umock_c/umock_c_prod.h"
+
+/*file acronyms: C = code, T = data Type. C is a decoration for code, T is a decoration for Data types. Having 2 decorations allows*/
```

Comment:
>C = code, T = data Type. C is a decoration for code, T is a decoration for Data types [](http://example.com/codeflow?start=17&length=85)

I don't know if I fully understand what "C" means from this comment...maybe an example or something would help or maybe I need to read more of the code.

Reply:
Maybe also "decoration" should be "alias" or "placeholder", but it's still not completely clear to me

Reply:
I read a bit further, maybe this would help me: "C = the type name to use for function names, T = the type name to use as a type" (or something like that.

Reply:
This is one of those comments that only creates chaos. Removed.